test/unit/yarn-workspaces/output.js

```diff
@@ -2,7 +2,6 @@
   "package.json",
   "test/unit/yarn-workspaces/input.js",
   "test/unit/yarn-workspaces/node_modules/x",
-  "test/unit/yarn-workspaces/node_modules/x/package.json",
```

Comment:
I believe this was incorrectly added in https://github.com/vercel/node-file-trace/pull/129/files#r455316015

Reply:
Agreed.